draft-ietf-rats-architecture.md

```diff
@@ -1188,64 +1188,80 @@ in terms of time(RX)-time(RG), then the Relying Party can check
+Handles are be qualifying data, such as nonces or signed timestamps.
+In this example, centrally generated signed timestamps are used as handles and are distributed in periodic intervals at time(HD).
+The distributor of handles includes a trustable source of time to produce the signed timestamps.
+This eliminates the need for clocks between the entities depicted in the diagram below. If entities include at least a relative clock, such as a tick counter, they can roughly determine (due to potentially latency during handle distribution) when the handles cannot be considered fresh anymore.
+Additionally, latency jitter for time(HD) between all entities involved (introduced by the conveyance of handles) is assumed to be the less than the time drift of a local clock with common precision.
 
 ~~~~
-.----------.         .---------------.              .----------.
-| Attester |         | Relying Party |              | Verifier |
-'----------'         '---------------'              '----------'
-  time(VG)                   |                           |
-        |                    |                           |
-     ---+----time(HD)--------+------time(HD))------------+---
-        |                    |                           |
-  time(AA)                   |                           |
-        |                    |                           |
-  time(EG)                   |                           |
-        |----Evidence------->|                           |
-        |    {time(EG)}   time(ER)--Evidence{time(EG)}-->|
-        |                    |                        time(RG)
-        |                 time(RA)<-Attestation Result---|
-        |                    |        {time(RX)}         |
-        ~                    ~                           ~
-        |                    |                           |
+                                                               .------------.
+                                                               | handle     |
+                                                               | distributor|
+                                                               '------------'
+.----------.         .---------------.              .----------.    | | |
+| Attester |         | Relying Party |              | Verifier |    | | |
+'----------'         '---------------'              '----------'    | | |
+  time(VG)                   |                           |          | | |
+        |                    |                           |          | | |
+        |                    |<-------handle------------------------/ | |
+        |                 time(HD)                       |            | |
+        |<-------handle-----------------------------------------------/ |
+  time(HD)                   |                           |<---handle----/
```

Comment:
this diagram is confusing since there are three time(HD) that all mean different things.
Suggest using ' and '' (prime and double prime) or something similar

Reply:
We tried to pick up on the notion that time(HD) is not exactly the same time, but "good enough" or "close enough" to the same time. If your time resolution is fine grained enough. The highlighted three time(HD) are different of course.

Again, the proposal to use three lines instead of one was made at the last call. If seeing this as ascii art makes us not using three lines in the end, but one. I am fine with that. We still have to highlight in text that there is some variance due to network conveyance here. This is how things work today - we simply need a way to depict that in an example here. I am open to suggestions and would be okay with "one line".